Applications/Spire/Source/KeyBindings/KeyBindings.cpp

```diff
@@ -18,6 +19,24 @@ bool KeyBindings::CancelActionBinding::operator !=(
   return !(*this == rhs);
 }
 
+KeyBindings KeyBindings::get_default_key_bindings() {
+  auto bindings = KeyBindings();
+  auto region = GetDefaultMarketDatabase().FromDisplayName("NSDQ");
```

Comment:
What is line 24 for? What is special about NSDQ?

Reply:
I was told originally that the default key bindings were the ones shown in the spec.

Reply:
Yes absolutely, can you point me to it for my own reference?

Reply:
Darryl told me through hangouts that the defaults for the task keys were what was in the table preview. The date for that message is March 27th, though, which I think was before the confusion about cancel key bindings and other defaults.

If there are other defaults documented somewhere I can't find them.

Reply:
Sure, very likely there was confusion. Replace line 24 with this:

    auto region = Region(Region::GlobalTag{});